.gitignore

```diff
@@ -14,7 +14,6 @@ artifacts/
 **/.vscode/
 
 cli/
-cli1.0.4/
```

Comment:
Please don't :)

If I switch between branches, this will break me. 

There's no harm in having this.

Reply:
stop living in the past, man :)